src/pages/library/DataBrowserDetails.js

```diff
@@ -0,0 +1,77 @@
+import { Fragment } from 'react'
+import { div, h, h1 } from 'react-hyperscript-helpers'
+import FooterWrapper from 'src/components/FooterWrapper'
+import { centeredSpinner, icon } from 'src/components/icons'
+import { libraryTopMatter } from 'src/components/library-common'
+
+
+const activeTab = 'browse & explore'
+const styles = {
+  page: { padding: 20, marginTop: 15 }
+}
+
+const getSnapshot = id => {
+  return {
+    namespace: 'harry-potter',
+    name: 'Harry Potter',
+    created: '2020-01-07T18:25:28.340Z',
+    lastUpdated: '2020-01-07T18:25:28.340Z',
+    tags: {
+      itemsType: 'AttributeValue',
+      items: ['1000 Genomes', 'CMG', 'Open Access', 'Exome']
+    },
+    description: 'The boy who lived',
+    lowerName: 'harry potter',
+    lowerDescription: 'lower description',
+    project: {
+      id: '112',
+      name: 'CDC'
+    },
+    subjects: 10,
+    dataType: 'RNA Seq',
+    keepCollapsed: true,
+    locked: false,
+    files: 15,
+    fileSize: 4432
+  }
+}
+
+const DataBrowserDetails = routeParams => {
+  const id = routeParams.id
+  const snapshot = getSnapshot(id)
+
+  return h(FooterWrapper, { alwaysShow: true }, [
+    libraryTopMatter(activeTab),
+    !snapshot ?
```

Comment:
This will never be true. I generally like the idea of skeletons like this, so I would update this to use a promise fulfilled on a delay so you can see the case with the spinner.

Reply:
Done!